docs/timeseries/initializing.rst

```diff
@@ -0,0 +1,277 @@
+.. _timeseries-initializing:
+
+Creating time series
+***************************************
+
+.. |Time| replace:: :class:`~astropy.time.Time`
+.. |TimeDelta| replace:: :class:`~astropy.time.TimeDelta`
+.. |Table| replace:: :class:`~astropy.table.Table`
+.. |QTable| replace:: :class:`~astropy.table.Table`
+.. |TimeSeries| replace:: :class:`~astropy.timeseries.TimeSeries`
+.. |BinnedTimeSeries| replace:: :class:`~astropy.timeseries.BinnedTimeSeries`
+
+Initializing a simple time series
+=================================
+
+The first type of time series that we will look at here is |TimeSeries|,
+which can be used for a time series which samples a continuous variable at
+discrete and instantaneous times. Initializing a |TimeSeries| can be done
```

Comment:
(minor)
```suggestion
discrete, instantaneous times. Initializing a |TimeSeries| can be done
```
?  Seems clearer to me because the "and" made it sound like the two go together rather than being distinct properties.